Makefile

```diff
@@ -1,3 +1,6 @@
+PUBLIC_REPO=mcr.microsoft.com/k8s/azureserviceoperator
+PLACEHOLDER_IMAGE=controller:latest
+
 # Image URL to use all building/pushing image targets
 IMG ?= controller:latest
```

Comment:
Should this be using `PLACEHOLDER_IMAGE` now?

Reply:
I've updated this.